scripts/run_driver_tests.psm1

```diff
@@ -471,7 +472,7 @@ function Invoke-CICDStressTests
 
     $LASTEXITCODE = 0
 
-    $TestCommand = "ebpf_stress_tests_km"
+    $TestCommand = ".\ebpf_stress_tests_km"
```

Comment:
Just checking - do we also need to specify '.exe' ? (I am not personally well versed enough in powershell to know, but I do see the other tests use it, for example $TestCommand = ".\xdp_tests.exe"